packages/js/product-editor/src/components/variations-table/variations-filter/variations-filter.tsx

```diff
@@ -181,6 +181,9 @@ export function VariationsFilter( {
 
 	const handleInputControlChange = useDebounce(
 		function handleInputControlChange( value: string | undefined ) {
+			if ( value === undefined ) {
+				return;
+			}
```

Comment:
I think we should use `value ?? ''` where `value` is used.

Reply:
Fixed with e1f833ffa51d6d0e7100adcce0347930fee5be5f